MagmaSafe.UseCases/Server/GetServerUseCase.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using MagmaSafe.Borders.UseCases;
+using MagmaSafe.Borders.Repositories;
+using MagmaSafe.Borders.Shared;
+using MagmaSafe.Shared.Models;
+
+namespace MagmaSafe.UseCases.Server {
+    public class GetServerUseCase : IGetServerUseCase {
+
+        private readonly IServersRepository serverRepository;
+
+        public GetServerUseCase(IServersRepository serverRepository) {
+            this.serverRepository = serverRepository;
+        }
+
+        public async Task<UseCaseResponse<Borders.Entities.Server>> Execute(string id) {
+            var response = new UseCaseResponse<Borders.Entities.Server>();
+
+            try {
+                var server = await serverRepository.GetById(id);
+
+                if (server != null) {
+                    return response.SetSuccess(server);
+                }
+                else {
+                    return response.SetNotFound(new ErrorMessage("00.01", $"Unable to find user with id = {id}"));
+                }
+            }
+            catch (Exception e) {
+                ErrorMessage errorMessage = new ErrorMessage("00.00", $"Unexpected error getting user from id: {id} \n Error: {e.Message}");
```

Comment:
Novamente: Não pode ter falando sobre **User**. Isso é um UseCase do Server.

```suggestion
                ErrorMessage errorMessage = new ErrorMessage("00.00", $"Unexpected error getting server from id: {id} \n Error: {e.Message}");
```